### DISCUSSION TITLE
select a slice in segmentation mask

### DISCUSSION THREAD

**Discussion by YingdiZhang** [Project-MONAI/MONAI#5086] [Q&A]
Thanks for the great work.
I get stuck in using MONAI. I aquired the segmentation mask of a CT volume, which is '1' for the foreground and '0' for background. and I would like to get the middle slice of foreground in saggital plane. How should I achive this?

**Comment by KumoLiu** [comment]
Hi, @YingdiZhang 

Thank you for using MONAI. I think `SpatialCrop` transform could help you solve your issue.
Suppose your segmentation mask is named `seg_mask`. Then you can do like :
```
center_idx = seg_mask.shape[2]//2  #  shape should be of (C,W,H,D)
center_slice = SpatialCrop(roi_slices=[slice(0, None), slice(center_idx, center_idx+1), slice(0, None)])(seg_mask)
```
